tests/features/person.feature

```diff
@@ -0,0 +1,172 @@
+@api
+Feature: Person content creation
```

Comment:
Let's create the behat context for person CT as well, based on what we have for other CTs.

Reply:
done